lib/handlers/quote-v2/schema.ts

```diff
@@ -0,0 +1,57 @@
+import Joi from 'joi';
+import { FieldValidator } from '../../util/validator';
+
+export * from './hard/schema';
+export * from './indicative/schema';
+
+/* v2 rfq request to filler */
+export const V2RfqRequestJoi = Joi.object({
+  tokenInChainId: FieldValidator.chainId.required(),
+  tokenOutChainId: FieldValidator.chainId.required(),
+  requestId: FieldValidator.uuid.required(),
+  tokenIn: Joi.string().required(),
+  amountIn: FieldValidator.amount.required(),
+  tokenOut: Joi.string().required(),
+  amountOut: FieldValidator.amount.required(),
+  quoteId: FieldValidator.uuid.required(),
+  numOutputs: Joi.number().min(1).required(),
+});
+
+export type V2RfqRequest = {
+  tokenInChainId: number;
+  tokenOutChainId: number;
+  requestId: string;
+  tokenIn: string;
+  tokenOut: string;
+  amount: string;
+  type: string;
+  quoteId: string;
+  numOutputs: number;
+};
+
+/* v2 rfq response from filler */
+export const V2RfqResponseJoi = Joi.object({
+  tokenInChainId: FieldValidator.chainId.required(),
+  tokenOutCHainId: FieldValidator.chainId.required(),
+  requestId: FieldValidator.uuid.required(),
+  tokenIn: Joi.string().required(),
+  amountIn: FieldValidator.amount.required(),
+  tokenOut: Joi.string().required(),
+  amountOut: FieldValidator.amount.required(),
+  cosigner: FieldValidator.address.required(),
```

Comment:
filler shouldn't need to return cosigner; I think we can actually keep the filler interface exactly the same as before?

Reply:
yeah I thought it could be helpful if we require them to explicit ack the labs address? but def not required